mantidproject/mantid
# CreateSampleWorkspace to also allow creating monitors

This adds the option "NumMonitors" to CreateSampleWorkspace.

**To test:**
- Run CreateSampleWorkspace with the NumMonitors and any a variety of other options, checking the monitors are added to the workspace (their workspace indices will be the after the detectors)
- Check the monitors are visible in the instrument view
- Check the updated documentation looks good

Fixes #17477 .

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
